app/models/project/Page.scala

```diff
@@ -120,6 +123,49 @@ case class Page(override val id: Option[Int] = None,
 
 object Page {
 
+  private object ExternalLinkResolver {
```

Comment:
If you make this a class (in a new file) you can use DI to get an instance of OreConfig and then use DI in a controller to get an instance of this class.

Reply:
That would still mean rebuilding the `HtmlRenderer` for each invocation though. Don't think I can use DI in `private val (markdownParser, htmlRenderer) `